examples/kafka/pom.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  
+  <parent>
+    <groupId>org.apache.apex</groupId>
+    <artifactId>malhar-examples</artifactId>
+    <version>3.8.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>malhar-examples-kafka</artifactId>
+  <packaging>jar</packaging>
+
+  <!-- change these to the appropriate values -->
+  <name>Apache Apex Malhar Kafka examples</name>
+  <description>
+    kafka2hdfs is a example Use of New Kafka Input Operator.
+    hdfs2kafka is a simple application to transfer data from HDFS to Kafka
+  </description>
+  
+  <dependencies>
+    <!-- add your dependencies here -->
+    <dependency>
+      <groupId>org.apache.apex</groupId>
+      <artifactId>malhar-kafka</artifactId>
+      <version>3.6.0</version> <!--check with pramod once-->
+      <!--version>${project.parent.version}</version-->
+      <exclusions>
+        <exclusion>
```

Comment:
@tweise I was doing some reading to understand why this was done. Looks like kafka-clients is the new java implementation for the kafka client API and kafka_<scala_version> is the scala client which we have been using in the older operators for some time.

I can only venture a guess, maybe the author of new kafka operator wanted to make sure that they are API compatible with java API and possibly switch completely to java API and hence they included kafka-clients as a dependency in the kafka operator project pom.xml. However, the author may have found that the java clients don't work correctly for all the functionality that we are using and hence, for example, they are using the scala client for the operator tests, as evidenced by the test dependency pointing to the scala client. I believe the original author of this example also is following the same pattern. They excluded the java client and included the scala one in the dependencies.